notebooks/llmu/readme.md

```diff
@@ -0,0 +1,37 @@
+# LLM University
+
+Here you will find the notebooks companion to the [LLM University course](https://llm.university/). It starts with the basics and progresses into more advanced topics. If you are new to Cohere, these guides are a great way to get started.
+
+The recommended approach is to go through the course and follow along in these notebooks. Having said that, you can still get a reasonable level of understanding if you only have time to go through the notebooks.
+
+# Contents
+
+## Text Representation
```

Comment:
Can we link to the Module (not necessarily every lesson, but just to the modules in LLMU)?

So something like:

## [Text Representation](https://docs.cohere.com/docs/intro-text-representation)